server/src/main/java/org/candlepin/audit/EventFactory.java

```diff
@@ -233,7 +233,7 @@ public Event subscriptionExpired(Subscription subscription) {
     public Event complianceCreated(Consumer consumer,
         Set<Entitlement> entitlements, ComplianceStatus compliance) {
         return new Event(Event.Type.CREATED, Event.Target.COMPLIANCE,
-            consumer.getName(), principalProvider.get(), consumer.getOwner().getId(), consumer.getId(),
+            consumer.getName(), principalProvider.get(), consumer.getOwner().getId(), consumer.getUuid(),
             consumer.getId(), null, buildComplianceDataJson(consumer, entitlements, compliance), null,
```

Comment:
We'll need the second usage of consumer.getId() replaced with consumer.getUuid() as well.